examples/find-element-by-trimmed-text/README.md

```diff
@@ -0,0 +1,9 @@
+# Find Element By Trimmed Text
+
+**Test Code**: [index.js](index.js)
+
+This example shows how to find the element on the page by trimmed text.
+
+`findElementByTrimmedText` is a TestCafe Selector that takes two arguments: `baseCSSSelector` and `text`. This selector runs a function on the client-side. The function locates the element on the page by a CSS selector, trims the inner text of the element, and returns the element if the `text` parameter matches the trimmed text.
```

Comment:
Change the last sentence to:
The function locates an element on a page by CSS selector, trims the inner text of that element, and returns the element if the `text` parameter matches the trimmed text.